### DISCUSSION TITLE
Power Quansheng UV-K5

### DISCUSSION THREAD

**Discussion by PU1VAR** [community/community#77289] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

For me it is the best FW for an independent programmer. The only thing missing is the power level adjustment. Example programmable LOW 1W, MID 2.5W and HI 5W.

**Comment by rostermann67** [comment]
Good afternoon. I would like to know if the only firmware that exploits the si4732 mod is CEC 0.51....and would it be possible to have a CEC 0.51 firmware version with spectrum analyzer...
